tests/utils.go

```diff
@@ -1185,6 +1186,7 @@ func LoggedInAlpineExpecter(vmi *v1.VirtualMachineInstance) (expect.Expecter, er
 	res, err := expecter.ExpectBatch(b, 180*time.Second)
 	if err != nil {
 		log.DefaultLogger().Object(vmi).Infof("Login: %v", res)
+		return nil, err
```

Comment:
You will have to close the expecter here.

Reply:
Oh did it under the previous commit, but did reset and forgot about it